### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by xlc**
Hoping this can improve https://github.com/AcalaNetwork/bodhi.js/issues/374
It is breaking the waffle examples

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -155,4 +155,4 @@ services:
       dockerfile: docker/truffle-tutorials-test.Dockerfile
       target: truffle-tutorials
     depends_on:
-      - eth-rpc-adapter-server
+      - eth-rpc-adapter-server-with-subql
```
Comment (shunjizhan) [inline docker-compose.yml:158]: why does truffle tutorials need subql?

Comment (zjb0807) [reply docker-compose.yml:158]: https://github.com/AcalaNetwork/truffle-tutorials/blob/d03704a28960522cd92044632a597dcb7719020e/token/test/token.js#L243-L273
This needs to query event

Comment (zjb0807) [reply docker-compose.yml:158]: https://github.com/AcalaNetwork/bodhi.js/runs/6054027642?check_suite_focus=true#step:6:4399

Comment (shunjizhan) [reply docker-compose.yml:158]: I think that's the logs from `transactionRceipt`, isn't it? We will only need subql when calling `eth_getlogs` explicitly, I believe the logs from transactionReceipt are from `Baseprovider.getTransactionReceiptAtBlock`

Comment (shunjizhan) [reply docker-compose.yml:158]: I do see the error msg `missing subql url to fetch logs` from `getLogs` though :thinking_face:

Comment (zjb0807) [reply docker-compose.yml:158]: common/config/rush/pnpm-lock.yaml
```
  ../../examples/hardhat-tutorials/token:
    specifiers:
      '@acala-network/eth-providers': ^2.2.3
      '@nomiclabs/hardhat-ethers': ^2.0.2
      '@nomiclabs/hardhat-waffle': ^2.0.1
      '@openzeppelin/contracts': ^4.4.0
      '@polkadot/api': ~7.14.4-6
      chai: ^4.3.4
      ethereum-waffle: ^3.4.0
      ethers: ~5.5.4
      hardhat: ^2.6.8
    dependencies:
      hardhat: 2.8.0
    devDependencies:
      '@acala-network/eth-providers': 2.2.3
      '@nomiclabs/hardhat-ethers': 2.0.3_ethers@5.5.4+hardhat@2.8.0
      '@nomiclabs/hardhat-waffle': 2.0.1_6a0d201cbcce3c9d971f3b7bc77e6f97
      '@openzeppelin/contracts': 4.4.2
      '@polkadot/api': 7.14.4-6
      chai: 4.3.4
      ethereum-waffle: 3.4.0
      ethers: 5.5.4
```

**Pull request closed without merging**